rasn-compiler-derive/Cargo.toml

```diff
@@ -16,6 +16,7 @@ authors = ["Kevin Westphal"]
 proc-macro = true
 
 [dependencies]
-rasn-compiler = { version = "0.3.0" }
+#rasn-compiler = { version = "0.3.0" }
+rasn-compiler = { path = "../rasn-compiler" }
```

Comment:
Could you change this to a dependency with multiple locations:
```toml
rasn-compiler = { path = "../rasn-compiler", version = "0.3.0" }
```

Reply:
:+1: